json/ollama.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "Ollama",
+    "slug": "ollama",
+    "categories": [
+        0
+    ],
+    "date_created": "2024-10-27",
+    "type": "ct",
+    "updateable": true,
+    "privileged": false,
+    "interface_port": "11434",
+    "documentation": "https://github.com/ollama/ollama/blob/main/docs/README.md",
+    "website": "https://ollama.com",
+    "logo": "https://github.com/ollama/ollama/assets/3325447/0d0b44e2-8f4a-4e99-9b52-a5c1c741c8f7",
+    "description": "Get up and running with large language models (LLMs). Run Llama 3.2, Phi 3, Mistral, Gemma 2, and other models. This script will install libraries for Intel GPU hardware acceleration.",
+    "install_methods": [
+        {
+            "type": "default",
+            "script": "ct/ollama.sh",
+            "resources": {
+                "cpu": "4",
+                "ram": "4096",
+                "hdd": "24",
+                "os": "ubuntu",
+                "version": "22.04"
+            }
+        }
+    ],
+    "default_credentials": {
+        "username": null,
+        "password": null
+    },
+    "notes": [
+        {
+            "text": "Due to the size of the models, 24GB of storage is required at a minimum.",
```

Comment:
```suggestion
```
The script does not install any models, It is the application and dependencies that requires this size

Reply:
I will add a note saying additional storage is required for models. 

But why does the script allocate 24GB of disk space, when only 11GB is used on a fresh install?

Reply:
I think mine went up to 19 gb at one point, but i was testing many different things so maybe 24gb is not actually needed